### PR TITLE
build: Add Release Engineering OWNERS

### PR DESCRIPTION
**Pull request by justaugustus**
Signed-off-by: Stephen Augustus <saugustus@vmware.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add Release Engineering subproject owners (SIG Chairs + Patch Release Team) as reviewers for some artifact-generating directories in `build/`.

**Which issue(s) this PR fixes**:

Related: #80546

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/cc @kubernetes/release-engineering 
/milestone v1.16
/priority important-soon

### CODE REVIEW

File: OWNERS_ALIASES
Comment (nikhita) [inline OWNERS_ALIASES]: Maybe I missed it, but looks like `sig-release-approvers` is not being used anywhere?

Comment (justaugustus) [reply OWNERS_ALIASES]: @nikhita -- it's not currently used anywhere, but we may decide to later.

File: OWNERS_ALIASES
Comment (liggitt) [inline OWNERS_ALIASES]: space needed before `#`?

File: build/debs/OWNERS
Comment (liggitt) [inline build/debs/OWNERS]: is there a reason this isn't under `approvers:`?

File: build/rpms/OWNERS
Comment (liggitt) [inline build/rpms/OWNERS]: same question about this being under `approvers:`?

**Comment by justaugustus** [comment]
@liggitt -- Fixed! Thanks for catching those. :)

**Comment by liggitt** [comment]
/approve
for new top-level aliases

/assign timothysc
for additions to existing package approver lists

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 
/hold 
^ for comments, feel free to unhold if needed.

**Comment by dims** [comment]
LGTM 👍

**Comment by justaugustus** [comment]
Thanks everyone! :)
/hold cancel

**Pull request merged**
